src/Text/Mustache/Compile.hs

```diff
@@ -22,7 +22,7 @@ import           Control.Monad.Except
 import           Control.Monad.State
 import           Data.Bool
 import           Data.HashMap.Strict        as HM
-import           Data.Text                  hiding (concat, find, map, uncons)
+import           Data.Text                  hiding (concat, find, map, show, uncons)
```

Comment:
This results in a warning with `text < 2.1.2`, which can easily be prevented by using qualified imports.